src/io.c

```diff
@@ -179,6 +179,83 @@ readFileFromTar(mtar_t* tar, char* path, size_t* lengthPtr, char** data) {
   return err;
 }
 
+internal bool
+fileExistsInTar(mtar_t* tar, char* path) {
```

Comment:
This and `directoryExistsInTar` might be useful in the future, but they aren't used currently, so we should remove them. We can always add them back later.

Reply:
Removed the 2 functions